README.md

```diff
@@ -1,3 +1,7 @@
+# Maintenance
+
+:heavy_exclamation_mark: This project is released as-is, and is not actively maintained by Civis Analytics.
```

Comment:
technically, there should not be a comment in this sentence since what's after the comma doesn't have a subject (e.g., "it").

Reply:
This is why I miss your code reviews, @mheilman 😉